test/testdata/namer/redefines_module_as_static_field.rb

```diff
@@ -0,0 +1,40 @@
+# typed: true
+# disable-stress-incremental: true
```

Comment:
I disabled stress incremental here because this test currently defines two symbols during incremental namer. I'm not exactly sure how; we end up with a `T::AbstractUtils::Methods::CallValidation` class during incremental resolver without `CallValidation` or `Modes` fields.